spec/support/vcr_cassettes/vaos/v2/appointments/get_appointments_200_booked_cerner.yml

```diff
@@ -57,7 +57,7 @@ http_interactions:
       string: '{"data":[{"id":"159472","identifier":[{"system":"urn:va.gov:masv2:cerner:appointment","value":"523938333130383130"}],"kind":"clinic","status":"booked","serviceType":"socialWork","serviceTypes":[{"coding":[{"system":"http://veteran.apps.va.gov/terminologies/fhir/CodeSystem/vats-service-type","code":"socialWork"}]}],
         "patientIcn":"1012845331V153043","locationId":"983","start":"2023-02-04T00:00:00Z","created":"2023-02-03T00:00:00Z","requestedPeriods":[{"start":"2023-02-04T00:00:00Z","end":"2023-02-04T00:00:00Z"}],"cancellable":true},{"id":"159473","identifier":[{"system":"http://vista-scheduling-provider-v1.sqa/vistasp/v1/Appointment/","value":"523938333130383130"}],"kind":"clinic","status":"proposed","serviceType":"covid","serviceTypes":[{"coding":[{"system":"http://veteran.apps.va.gov/terminologies/fhir/CodeSystem/vats-service-type","code":"primaryCare"}]}],"serviceCategory":[{"coding":[{"system":"http://www.va.gov/Terminology/VistADefinedTerms/409_1","code":"REGULAR",
         "display":"REGULAR"}],"text":"REGULAR"}],
-        "patientIcn":"1012845331V153043","locationId":"983","start":"2023-02-04T00:00:00Z","created":"2023-02-03T00:00:00Z","requestedPeriods":[{"start":"2023-02-04T00:00:00Z","end":"2023-02-04T00:00:00Z"}],"cancellable":true}]}'
+        "patientIcn":"1012845331V153043","locationId":"983","start":"2023-02-04T00:00:00Z","created":"2021-09-01T00:00:00Z","requestedPeriods":[{"start":"2023-02-04T00:00:00Z","end":"2023-02-04T00:00:00Z"}],"cancellable":true}]}'
```

Comment:
The presentation filter depends on a `created` attribute existing for `proposed` and `cancelled` appointments. I added this field to all appointments in the cassettes that fit this criteria